Someday, when Solaris has GNU getopts, we won't need this kludge.  This allows
test/refresh-z to pass where it wants to use -z without a name argument.

Index: quilt/refresh.in
===================================================================
--- quilt/refresh.in
+++ quilt/refresh.in
@@ -106,7 +106,12 @@ do
 	-z)
 		opt_fork=1
 		opt_new_name=$2
-		shift 2 ;;
+		shift 2
+		if [ "$opt_new_name" = "--" ]
+		then
+			opt_new_name=
+			break;
+		fi ;;
 	-h)
 		usage -h ;;
 	--no-timestamps)
